tools/scxtop/Cargo.toml

```diff
@@ -56,6 +56,11 @@ log-panics = { version = "2", features = ["with-backtrace"]}
 hashbrown = "0.15.2"
 smartstring = { version = "1.0.1", features = ["serde"] }
 nix = { version = "0.29", features = ["time"] }
+criterion = "0.6.0"
```

Comment:
nit: I think you can put this into [`dev-dependencies`](https://doc.rust-lang.org/rust-by-example/testing/dev_dependencies.html) since the binary shouldn't need criterion for runtime.